docs/cmdline-opts/override.d

```diff
@@ -0,0 +1,5 @@
+Long: override
+Help: Permit/Prohibit overwriting of files
+---
+If you route the output to a file which already exists, it can be disabled using
+--no-override. The --override argument is the default.
```

Comment:
Can I suggest a clearer wording that focuses on the use and the effects? Maybe something like this:

"This option allows curl to overwrite already existing files with the output options. If --no-overwrite is used, curl will instead return an error (which?) rather than overwriting an existing file. The default is to allow overwrites."